tempesta_fw/t/unit/test_sched_ratio.c

```diff
@@ -0,0 +1,350 @@
+/**
+ *		Tempesta FW
+ *
+ * Copyright (C) 2014 NatSys Lab. (info@natsys-lab.com).
+ * Copyright (C) 2015-2017 Tempesta Technologies, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License,
+ * or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59
+ * Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ */
+#include <asm/fpu/api.h>
+
+#undef tfw_sock_srv_init
+#define tfw_sock_srv_init test_ratio_sock_srv_conn_init
+#undef tfw_sock_srv_exit
+#define tfw_sock_srv_exit test_ratio_sock_srv_exit
+#undef tfw_srv_conn_release
+#define tfw_srv_conn_release test_ratio_srv_conn_release
+#undef tfw_sock_srv_cfg_mod
+#define tfw_sock_srv_cfg_mod test_ratio_srv_cfg_mod
+
+#include "sock_srv.c"
+
+#ifdef module_init
+#undef module_init
+#undef module_exit
+#define module_init(func)
+#define module_exit(func)
+#endif
+
+#include "../../sched/tfw_sched_ratio.c"
```

Comment:
Yes, that was the reason I didn't add unit tests for the new functionality. I didn't feel like they would be useful enough. These kind of tests would better be done with functional testing.

I will open a task for this, thanks.